src/components/structures/RoomSubList.js

```diff
@@ -344,7 +344,7 @@ const RoomSubList = React.createClass({
                 "mx_RoomSubList_nonEmpty": len && !this.state.hidden,
             });
             if (this.state.hidden) {
-                return <div className={subListClasses}>
+                return <div className={subListClasses} style={{flexBasis: "unset", flexGrow: "unset"}}>
```

Comment:
is there a reason for doing this inline?

Reply:
as an inline style you mean? Yeah, room sub lists are sized manually (by dragging the handles) or initially based on the amound of tiles they have by setting their flexGrow style property. When collapsing, this needs to be reset, and hence also needs to be applied to the inline style property.

Or you just meant doing it inline in the code here? In that case I did it because there was nothing variable about it, but happy to change it.

Reply:
@dbkr Thanks for looking at all the PRs, just this comment remaining here.